packages/web-components/src/components/va-checkbox/va-checkbox.scss

```diff
@@ -111,3 +110,10 @@
   margin-top: 0 !important;
   margin-bottom: 1.2rem !important;
 }
+
+:host(:not([uswds])[tile='true']) label {
+  padding: 12px 16px 12px 12px;
+  border: 2px solid var(--color-gray-light);
+  border-radius: 0.25em;
+  width: calc(100% - 16px);
```

Comment:
I think that we want the checkbox tile to look the same as the [radio tile](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-radio-option/va-radio-option.scss#L75-L84). The width specifically. I'm not sure why the width calc is needed.

**v1 Radio Tile**

![Screenshot 2023-04-24 at 9 08 09 AM](https://user-images.githubusercontent.com/872479/234021573-c5c782a2-cf97-405a-8474-aad778a17220.png)

**v1 Checkbox Tile**

![Screenshot 2023-04-24 at 9 07 58 AM](https://user-images.githubusercontent.com/872479/234021575-0598364b-ffe3-4bb9-b4e3-29c2d2e979ce.png)

It would also be nice to see the `va-checkbox` tile story to include hint text so we can see how it all looks together with the tile (like radio has).

Reply:
Added the in-tile hint text in USWDS, looks like v1 doesn't have that feature.

Adjusted width of tile